<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>triggerdialog</class>
 <widget class="QDialog" name="triggerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger setup</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>14</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comLevelCond">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>When level A is satisfied</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>When level B is satisfied</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>When level A and level B are satisfied</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>When level A or level B are satisfied</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>681</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Level A conditions:</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>652</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="chkLevAValEnable">
     <property name="text">
      <string>Value of</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD0">
     <property name="text">
      <string>D0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD1">
     <property name="text">
      <string>D1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD2">
     <property name="text">
      <string>D2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD3">
     <property name="text">
      <string>D3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD4">
     <property name="text">
      <string>D4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD5">
     <property name="text">
      <string>D5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD6">
     <property name="text">
      <string>D6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevAD7">
     <property name="text">
      <string>D7</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comLevAValueCond">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not in range</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ediLevAValueStart">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labLevAValTo">
     <property name="text">
      <string> To </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ediLevAValueEnd">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>146</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Edge A occurs</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spiLevATimes">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>time(s)</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>353</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="chkLevAIDEnable">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comLevAIDCond">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not in range</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ediLevAIDStart">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="labLevAIDTo">
     <property name="text">
      <string> To </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ediLevAIDEnd">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>681</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Level B conditions:</string>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>652</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="chkLevBValEnable">
     <property name="text">
      <string>Value of</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD0">
     <property name="text">
      <string>D0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD1">
     <property name="text">
      <string>D1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD2">
     <property name="text">
      <string>D2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD3">
     <property name="text">
      <string>D3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD4">
     <property name="text">
      <string>D4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD5">
     <property name="text">
      <string>D5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD6">
     <property name="text">
      <string>D6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLevBD7">
     <property name="text">
      <string>D7</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comLevBValueCond">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not in range</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ediLevBValueStart">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labLevBValTo">
     <property name="text">
      <string> To </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ediLevBValueEnd">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>30</y>
      <width>146</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Edge B occurs</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spiLevBTimes">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>time(s)</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>353</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="chkLevBIDEnable">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comLevBIDCond">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equal to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not in range</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ediLevBIDStart">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="labLevBIDTo">
     <property name="text">
      <string> To </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ediLevBIDEnd">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Action at trigger event:</string>
   </property>
   <widget class="QComboBox" name="comAction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Show message</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enable log</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Disable log</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Play sound</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Execute command</string>
     </property>
    </item>
   </widget>
   <widget class="QSplitter" name="splitter_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="chkActDelayEnable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Delay before msec </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ediActDelay">
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="ediActionCommand">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
